app/policies/tag_group_policy.rb

```diff
@@ -0,0 +1,3 @@
+class TagGroupPolicy < StoryPolicy
+
```

Comment:
Need to think some policies methods here...

For me it's simple, if user is admin, then he can handle all tag groups.